src/modules/app/components/side-nav/side-nav.jsx

```diff
@@ -92,11 +98,28 @@ export default class SideNav extends Component {
                   onClick={linkClickHandler}
                 >
                   <Icon />
-                  <span className="item-title">{item.title}</span>
+                  <span className={Styles['item-title']}>{item.title}</span>
                 </Link>
               </li>
             )
           })}
+          {logged && mobile &&
+            <li
+              key="notifications"
+            >
+              <a
```

Comment:
For semantics sake, change this to a `button` since it’s only performing an action + not navigating.